src/developer/web-api/tracker.md

```diff
@@ -1399,14 +1400,15 @@ data element with a numeric type. In that case, the value is expected to be nume
 related to a mismatch between a type and a value will result in the same error code but with a
 specific message related to the type of violation.
 
-Mandatory attributes and data values are also checked. Currently, removing mandatory attributes is
-not allowed. Some use-cases require values to be sent separately, while others require all values to
+Mandatory attributes and data values are also checked on creation, on update mandatory attributes
+and data valuesare not required in the payload. Currently, removing mandatory attributes and data values is
+never allowed. Some use-cases require values to be sent separately, while others require all values to
 be sent as one. Programs can be configured to either validate mandatory attributes `ON_COMPLETE` or
 `ON_UPDATE_AND_INSERT` to accommodate these use-cases.
 
 The import will validate unique attributes at the time of import. That means as long as the provided
 value is unique for the attribute in the whole system, it will pass. However, if the unique value is
-fpound used by any other tracked entity other than the one being imported, it will fail.
+found used by any other tracked entity other than the one being imported, it will fail.
```

Comment:
```suggestion
found to be used by any other tracked entity other than the one being imported, it will fail.
```
maybe this is already fine, but I find the above a bit clearer